hibernate-core/src/main/java/org/hibernate/type/JaxbXmlFormatMapper.java

```diff
@@ -44,7 +44,6 @@
 public class JaxbXmlFormatMapper implements FormatMapper {
```

Comment:
Since you moved other classes to dedicated packages, what about moving this one here as well into e.g. `jakartaxml`?

Reply:
good idea, thanks